recipes/iris/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Make sure Iris can find the udunits2 library while it's in the temporary
+# build environment. This is necessary because the build process also compiles
+# the pyke rules, which requires importing iris.
+if [[ $(uname) == Darwin ]]; then
+    EXT=dylib
+else
+    EXT=so
+fi
+
+SITECFG=lib/iris/etc/site.cfg
+echo "[System]" > $SITECFG
+echo "udunits2_path = $PREFIX/lib/libudunits2.${EXT}" >> $SITECFG
+
+rm -rf lib/iris/tests/results lib/iris/tests/*.npz
+
```

Comment:
eventually I'd like to stop building the unpack extension, and just depend on mo_pack.